gtsfm/configs/deep_front_end.yaml

```diff
@@ -13,14 +13,18 @@ SceneOptimizer:
 
   two_view_estimator:
     _target_: gtsfm.two_view_estimator.TwoViewEstimator
-    corr_metric_dist_threshold: 3 # in px
+    eval_threshold_px: 3 # in px
+    estimation_threshold_px: 4 # for homography estimator
```

Comment:
why just for homography estimation? What about epipolar geometry cases? Also is there a better name than `estimation_threshold_px`?

Reply:
This homography based check should be verifier specific? Because in the future we might have a deep-learning based verifier which performs the model selection via networks too?

Reply:
I don’t think estimation_threshold_px is a great name either. How is estimation measured in px?

Reply:
So technically epipolar distance (squared Sampson) is measured in pixels (or pixels^2).

But unfortunately we measure epipolar distance twice -- once in estimation, once in evaluation. So we need to differentiate between the two.

Reply:
How about `estimation_error_threshold_px` and `eval_error_threshold_px`?

Reply:
Thanks for the suggestions. I worry that those names are a bit too longwinded, though.

Reply:
> why just for homography estimation? What about epipolar geometry cases? Also is there a better name than `estimation_threshold_px`?

if we want to compare the # homography inliers vs. # E or F inliers in the future, like COLMAP does, then we would want to share the same threshold between homography and E/F fitting
4:21
that would mean that estimation threshold would need to be passed to the TwoViewEstimator, since we’ve defined our verifiers to operate on E/F
4:21
that would mean that we would need to pass that threshold to the verify() method from the TwoViewEstimator every time, or provide the same threshold in two places — the TwoViewEstimator config options, and the Verifier config options

Reply:
resolved by using `self.verifier._estimation_threshold_px` within TwoViewEstimator, for homography, in the future